### PR TITLE
MRG: Enable fixing of mag channels

### PR DESCRIPTION
**Pull request by Eric89GXL**
Addresses part of #2502.

Ready for review/merge from my end.

### CODE REVIEW

File: mne/channels/channels.py [hunk 1]
```diff
@@ -744,3 +744,39 @@ def _ch_neighbor_connectivity(ch_names, neighbors):
...
+
+    Notes
+    -----
+    This function changes magnetometer coil types 3022 (T1: SQ20483N) and
+    3023 (T2: SQ20483-A) to 3024 (T3: SQ20950N) in the channel definition
+    records in the info structure.
+
+    Neuromag Vectorview systems can contain magnetometers with two
+    different coil sizes (3022 and 3023 vs. 3024). The systems
+    incorporating coils of type 3024 were introduced last. At some sites
```
Comment (wronk) [inline mne/channels/channels.py]: From Samu/Jukka, T3 coil types are used on most Neuromag systems, so maybe add that detail. Something like " ... were introduced last and are used at the majority of MEG sites."

**Comment by wronk** [comment]
Other than 1 comment above, LGTM

**Comment by agramfort** [comment]
@mshamalainen any thoughts?

**Comment by Eric89GXL** [comment]
@agramfort FYI the verbiage in the `Notes` section was adapted from Matti's C function `mne_fix_mag_coil_types`, so hopefully we won't see too much objection from @mshamalainen :)

**Comment by Eric89GXL** [comment]
@wronk good call, comment addressed.

**Comment by agramfort** [comment]
merging then

thanks @Eric89GXL

**Pull request merged**